ghdocs/components/PersonaCard.md

```diff
@@ -22,10 +22,10 @@ The visualization of details of an individual including Skype contact details, e
 
 <pre>
     <code>
- &lt;!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. --&gt;
+ 
```

Comment:
There are extra lines and whitespace when viewing in Markdown and regular docs as well as formatting issues with the code snippets.